package-lock.json

```diff
@@ -20742,7 +21065,7 @@
 				},
 				"node-pre-gyp": {
 					"version": "0.12.0",
-					"resolved": "https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.12.0.tgz",
+					"resolved": false,
```

Comment:
!

https://github.com/WordPress/gutenberg/issues/16157#issuecomment-601187274

#21306

Reply:
Let's do it as the last step because it's difficult to apply changes to the lock file with all rebases necessary 😞 

I'm sure I have this field set to `false` in all PRs I opened this week that change the lock file.

Reply:
It should be good now :)